src/app/api.service.ts

```diff
@@ -10,7 +10,14 @@ export class ApiService {
   private BASE_URL = "http://localhost:8080/carapp";
```

Comment:
Instead of having this value hardcode here, try to use environment variables:
https://medium.com/@balramchavan/configure-and-build-angular-application-for-different-environments-7e94a3c0af23